### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NukeManDan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ A set of tools for writing [Substrate](https://substrate.io/)-based
 [Polkadot](https://wiki.polkadot.network/en/)
 [parachains](https://wiki.polkadot.network/docs/en/learn-parachains). Refer to the included
 [overview](docs/overview.md) for architectural details, and the
-[Cumulus tutorial](https://docs.substrate.io/tutorials/v3/cumulus/start-relay) for a
+[Connect to other chains tutorials](https://docs.substrate.io/tutorials/connect-other-chains/) for a
```
Comment (gilescope) [inline README.md:14]: that link 404s for me?

Comment (NukeManDan) [reply README.md:14]: Just changed https://docs.substrate.io/tutorials/connect-relay-and-parachains/

Comment (NukeManDan) [reply README.md:14]:
```suggestion
[Connect to relay and  parachain tutorials](https://docs.substrate.io/tutorials/connect-relay-and-parachains/) for a
```

Comment (NukeManDan) [reply README.md:14]: @lisa-parity could we put in a redirect for changes like this as I am sure the previous URL is used many places external to the docs too now.

Comment (lisa-parity) [reply README.md:14]: yeah, that makes sense.

**Review by gilescope — APPROVED** [review]

**Review by muharem — APPROVED** [review]

**Comment by gilescope** [comment]
bot merge

**Pull request merged**